src/F0_output.py

```diff
@@ -3,11 +3,32 @@
 import pandas as pd
 import logging
 import matplotlib.pyplot as plt
+import os
 
 import src.F1_plotting as plots
 
+r"""
+Module F0 Output
+================
+The model F0 output defines all functions that store evaluation results to file.
+- Aggregate demand profiles to a total demand profile
+- Plotting all energy flows for both 14 and 365 days for each energy bus
+- Store timeseries of all energy flows to excel (one sheet = one energy bus)
+- execute function: plot optimised capacities as a barchart (F1)
+- execute function: plot all annuities as a barchart (F1)
+- store scalars/KPI to excel
+- process dictionary so that it can be stored to Json
+- store dictionary to Json
+"""
+
 
 def evaluate_dict(dict_values):
+    """
```

Comment:
I think this is not the docstring format we agreed to use with @SabineHaas https://github.com/rl-institut/mvs_eland/issues/155#issuecomment-594652686 I have been using the same format as @SabineHaas  on my end